File: Python/calculadora.py
def calculadora(num1,num2,operação):
    if operação == 1:
        soma = num1 + num2
        return soma
    elif operação == 2:
        subtracao = num1 - num2
        return subtracao
    elif operação == 3:
        multiplicacao = num1 * num2
        return multiplicacao
    elif operação == 4:
        divisao = num1 / num2
        return divisao
    else:
        return 0

File: JavaScript/calculadora.js
var num1 = Number(prompt("Digite um número: "));
var operador = (prompt("Digite o sinal a operação que deseja realizar: "))
var num2 = Number(prompt("Digite outro número: "));

var soma = num1+num2;
var sub = num1-num2;
var mult = num1*num2;
var div = num1/num2;

if (operador == ("+")) {
    console.log(num1, "+", num2, "=", soma);
}

else if (operador == ("-")) {
    console.log(num1, "-", num2, "=", sub);
}

else if (operador == ("x")) {
    console.log(num1, "x", num2, "=", mult);
}

else if (operador == ("/")) {
    console.log(num1, "dividido por", num2, "=", div);
}

else {
    console.log("Operação Inválida");
}
